src/etc/inc/system.inc

```diff
@@ -186,6 +186,11 @@ function system_resolvconf_generate($verbose = false)
         $resolvconf .= "nameserver 127.0.0.1\n";
     }
 
+    if (isset($syscfg['domainsearchlist'])) {
+        /* add custom search list to search list */
+        $search = array_merge($search, explode(' ',$syscfg['domainsearchlist']));
```

Comment:
The DHCPv4 services uses semicolon to separate domain names (see /services_dhcp.php). We should aim in consistency.
```suggestion
        $search = array_merge($search, explode(';', $syscfg['domainsearchlist']));
```

Reply:
I would rather see a tokenizer to hide the complexity, but this one uses a "," to separate. It could be changed, but it doesn't make much sense either way if we aim to hide the presence of the separator more and let it work like all other tokenizers in the GUI?

Reply:
> I would rather see a tokenizer to hide the complexity

:+1: Definitely the best idea. How's it realized in the backend (space vs. comma vs. semicolon) is not important here, I'd say. I just find it confusing when one GUI element uses this and another GUI element that. With the tokenizer this difference is gone anyway.